src/Row.tsx

```diff
@@ -109,8 +103,6 @@ function Row<R, SR>(
   );
 }
 
-export default memo(Row) as <R, SR>(props: RowRendererProps<R, SR>) => JSX.Element;
```

Comment:
Without `forwardRef`, `ref` is set to `{}` and it fails at runtime. Previously `useCombinedRefs` was handling it